lib/profiles.js

```diff
@@ -84,6 +83,7 @@ module.exports.create = async ({
   // DID#KEY_IDENTIFIER
   const profileDid = key.id.split('#')[0];
   profileKeystore.controller = profileDid;
+  profileKeystore.referenceId = 'primary';
```

Comment:
Are we using this reference ID? If not, we should remove it.

Reply:
First I want to point out that this is not new code, but it has been moved chronologically to avoid `DuplicateError` on the controller/referenceId compound index on keyStore collection in bedrock-kms.

That said, this is the being marked as the `primary` keyStore for the profile.

I'm not sure how `referenceId === 'primary'` is used downstream.

@JSAssassin We can leave this comment here for now and see if we can answer that question as this code is propagated downstream.

Reply:
@gannan08 can you provide insights into if/how `referenceId === primary` is used downstream?

Reply:
@mattcollier I could not find any usage of that `referenceId` in bedrock-web-profile-manager. It may be used elsewhere, but was unsuccessful figuring out exactly where.

Reply:
OK, per our offline discussion...  This line should be removed (referenceID will no longer be set) as part of this breaking change and should be listed explicitly among the breaking changes.

I believe there's a unit test here or a downstream suite that tests for the current behavior and will need to be updated.

@JSAssassin one more change here to include in this breaking release.

Reply:
Addressed here aff2327